src/jaegertracing/TracerTest.cpp

```diff
@@ -122,16 +122,15 @@ TEST(Tracer, testTracer)
     const auto tracer =
         std::static_pointer_cast<Tracer>(opentracing::Tracer::Global());
 
-    auto tagItr =
-        std::find_if(std::begin(tracer->tags()),
-                     std::end(tracer->tags()),
-                     [](const Tag& tag) {
-                         return tag.key() == kJaegerClientVersionTagKey;
-                     });
+    auto tagItr = std::find_if(
+        std::begin(tracer->tags()),
+        std::end(tracer->tags()),
+        [](const Tag& tag) { return tag.key() == kJaegerClientVersionTagKey; });
```

Comment:
Looks like you ran `clang-format` here. Please run `scripts/clang-format.sh` before pushing next commit to make sure all files are formatted correctly.

Reply:
Done.